### PR TITLE
PSP-6517 Compensation Requisition Export

### PR DESCRIPTION
**Pull request by asanchezr**
export_screen

excel_report

### CODE REVIEW

File: source/backend/api/Areas/Reports/Controllers/AcquisitionController.cs [hunk 1]
```diff
@@ -74,6 +79,49 @@ public IActionResult ExportLeases([FromBody] AcquisitionReportFilterModel filter
 
             return ReportHelper.GenerateExcel(reportAgreements, "Agreement Export");
         }
+
+        /// <summary>
+        /// Exports compensation requisitions as Excel file.
+        /// Include 'Accept' header to request the appropriate export -
+        ///     ["text/csv", "application/application/vnd.ms-excel"].
```
Comment (eddherrera) [inline source/backend/api/Areas/Reports/Controllers/AcquisitionController.cs:86]: nit: remove "text/csv" from summary.

File: source/backend/api/Areas/Reports/Controllers/AcquisitionController.cs [hunk 2]
```diff
@@ -74,6 +79,49 @@ public IActionResult ExportLeases([FromBody] AcquisitionReportFilterModel filter
...
+        /// </summary>
+        /// <param name="filter"></param>
+        /// <returns></returns>
```
Comment (eddherrera) [inline source/backend/api/Areas/Reports/Controllers/AcquisitionController.cs:89]: nit: add a return description for summary.

File: source/backend/api/Areas/Reports/Models/Acquisition/CompensationFinancialReportModel.cs [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+        [DisplayName("Total Expenditures shown in this report (Tax) - final")]
+        public decimal AllExpendituresFinalTaxAmount { get; set; }
+
+        [DisplayName("Total Expenditures shown in this report (Inc. Tax) - final")]
+        public decimal AllExpendituresFinalTotalAmount { get; set; }
+
+        public CompensationFinancialReportModel(PimsCompReqFinancial financial, CompensationFinancialReportTotalsModel reportTotals, ClaimsPrincipal user)
+        {
+            ExportBy = user.GetDisplayName();
+            ExportDate = DateTime.Now.ToString("dd/MM/yyyy");
```
Comment (eddherrera) [inline source/backend/api/Areas/Reports/Models/Acquisition/CompensationFinancialReportModel.cs:94]: could you confirm this date format? Is not the same we use trough the system.

Comment (asanchezr) [reply source/backend/api/Areas/Reports/Models/Acquisition/CompensationFinancialReportModel.cs:94]: I re-used the same pattern that @devinleighsmith implemented for his export but will ask the BAs for confirmation

File: source/backend/api/Areas/Reports/Controllers/AcquisitionController.cs (hunk 1 above)
Comment (asanchezr) [reply source/backend/api/Areas/Reports/Controllers/AcquisitionController.cs]: removed

File: source/backend/api/Areas/Reports/Controllers/AcquisitionController.cs (hunk 2 above)
Comment (asanchezr) [reply source/backend/api/Areas/Reports/Controllers/AcquisitionController.cs]: updated

File: source/backend/api/Areas/Reports/Models/Acquisition/CompensationFinancialReportModel.cs (hunk 3 above)
Comment (asanchezr) [reply source/backend/api/Areas/Reports/Models/Acquisition/CompensationFinancialReportModel.cs]: Updated per BAs comment: 
> re: date format please use: yyyy-mm-dd

**Review by eddherrera — APPROVED** [review]

**Pull request merged**